<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#DEB6AB;</string>
  </property>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>431</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:12pt; background-color:whitesmoke; color:black;</string>
   </property>
  </widget>
  <widget class="QListWidget" name="tasksListWidget">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>150</y>
     <width>341</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:12pt;background-color:none;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>460</y>
     <width>341</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color:#AC7D88;
color:white;
font:11pt;</string>
   </property>
   <property name="text">
    <string>Değişiklikleri Kaydet</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>110</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color:#AC7D88;
color:white;
</string>
   </property>
   <property name="text">
    <string>Görev Ekle</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size : 24pt;
background : #85586F;
color:white;
</string>
   </property>
   <property name="text">
    <string>Ajanda Aracı</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="taskLineEdit">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>110</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:12pt;background-color:none;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
